firmware/controllers/date_stamp.h

```diff
@@ -1,3 +1,2 @@
-#ifndef VCS_DATE
-#define VCS_DATE 20200617
-#endif
\ No newline at end of file
+#pragma once
+#define VCS_DATE 20200618
```

Comment:
is this change on purpose?

Reply:
nope. set_date ran and I didn't notice.

Reply:
Won't hurt a thing though.

Reply:
did it run on your branch? perhaps it should only run when on real rusefi/master

Reply:
yep looks like that's exactly what happened

Reply:
good idea. I'll take care of that.